manifests/kustomize/base/cache-deployer/cluster/cache-deployer-clusterrolebinding.yaml

```diff
@@ -8,4 +8,5 @@ roleRef:
   name: kubeflow-pipelines-cache-deployer-clusterrole
 subjects:
 - kind: ServiceAccount
-  name: kubeflow-pipelines-cache-deployer-sa
\ No newline at end of file
+  name: kubeflow-pipelines-cache-deployer-sa
+  namespace: kubeflow
```

Comment:
I wasn't sure if namespace here can be overridden by kustomization.yaml's namespace.

Can you have a try? If override works, I think this is good

Reply:
Moving namepsace to kustomization.yaml doesn't work. We need namespace here.

Reply:
> the namespace should be a parameter instead of a hard coded one.

The namespace must be present as it is required by the schema. But `kustomize` will override it based on the namespace in the kustomization.yaml. See https://github.com/kubeflow/pipelines/pull/3376#discussion_r401837572

Reply:
Thanks! SGTM

Reply:
ah thanks @Ark-kun that's right. kustomize detects that. 
@rui5i could you add a short comment to call it out for better readability?